pkg/services/ngalert/state/template/template.go

```diff
@@ -51,59 +49,63 @@ func (v Value) String() string {
 	return strconv.FormatFloat(v.Value, 'f', -1, 64)
 }
 
-func NewValues(values map[string]eval.NumberValueCapture) map[string]Value {
-	m := make(map[string]Value)
-	for k, v := range values {
+func NewValues(caps map[string]eval.NumberValueCapture) map[string]Value {
```

Comment:
Wouldn't it be even more readable if we call it `captures`
```suggestion
func NewValues(captures map[string]eval.NumberValueCapture) map[string]Value {
```

Reply:
I'll keep it as `caps` just for now as we use `caps` in the eval package too!